frontend/src/metabase/visualizations/components/settings/ChartSettingInput.tsx

```diff
@@ -25,7 +26,7 @@ export const ChartSettingInput = ({
 
   const onChangeRef = useLatest(onChange);
   const onChangeDebounced = useMemo(
-    () => debounce((value: string) => onChangeRef.current(value), 400),
+    () => debounce((value: string) => onChangeRef.current(value), delay(400)),
```

Comment:
This already exists since https://github.com/metabase/metabase/pull/50806 which was built for loki, but why not do that for unit tests too. We genrally don't have to wait for a debounce in unit tests.

But I won't go around changing every delay to use this intermediary function as the change would be too big.

Reply:
I can see we're using `jest.useFakeTimers({ advanceTimers: true })` and/or `jest.advanceTimersByTime(...)` in the codebase to handle debounces in some tests. I wonder if that's better than skipping all debounces with `METABASE_REMOVE_DELAYS` or not